js/blocks/PitchBlocks.js

```diff
@@ -616,9 +609,8 @@ class TranspositionFactorBlock extends ValueBlock {
 class ConsonantStepSizeDownBlock extends ValueBlock {
     constructor() {
         //.TRANS: step down one note in current musical scale
-        super('consonantstepsizedown', _('scalar step down'));
+        super('constonantstepsizedown', _('scalar step down'));
```

Comment:
Why did you change the block internal name? This will break old projects.

Reply:
^^ that is the place where the name was changed.

Reply:
@walterbender Actually it's due to some commit history. This block was originally named here as `constonantstepsizedown` whereas in turtledef.js it was named `consonantstepsizedown` . So I corrected it in #2083 , commit 0838257bd88a9c12b789357cf3932b14f8b4217b
So I think it's some issue with git, @b18050 didn't change it explicitly.

Reply:
consonantstepsizedown is the correct name.

Reply:
Yes.